src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -220,5 +232,36 @@ public static void ValidateSslCertificate(HttpClient client)
                 );
             }
         }
+
+        /// <summary>
+        /// Format request information when LogLevel is set to debug
+        /// </summary>
+        ///
+        /// <param name="request">HTTP request</param>
+        private static void LogRequest(Request request)
+        {
+            Console.WriteLine("-- BEGIN Twilio API Request --");
+            Console.WriteLine("request.method: " + request.Method);
+            Console.WriteLine("request.URI: " + request.Uri);
+
+            if (request.QueryParams != null)
+            {
+                request.QueryParams.ForEach(kpv => Console.WriteLine(kpv.Key + ":" + kpv.Value));
```

Comment:
what does kpv stand for?

Reply:
It stands for the key pair value of each query param but if there is a different reference that makes this more readable I can change it.

Reply:
Ah I usually use the term _key value pair (kvp)_, so the ordering threw me off. Mmmm I'd lean towards a more descriptive variable name. Usually if I can intuit what it represents it's good, but in this case I had to ask. Maybe just `param` => `param.Key` and `param.Value`?